keps/sig-node/3066-subsecond-probes/kep.yaml

```diff
@@ -0,0 +1,30 @@
+title: Subsecond Probes
+kep-number: 3066
+authors:
+  - "@mikebrow"
+  - "@psschwei"
+  - "@MHBauer"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2021-11-30
+reviewers:
```

Comment:
please add @aojea or somebody he will recommend to review for scalability issues